srcpkgs/textadept/template

```diff
@@ -9,44 +9,46 @@ makedepends="gtk+3-devel ncurses-devel"
-distfiles="https://github.com/rgieseke/textadept/archive/textadept_${version}.tar.gz
- http://foicica.com/hg/scintilla/archive/99fa62b828ee.zip
- http://www.lua.org/ftp/lua-5.3.5.tar.gz
- http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz
- http://github.com/keplerproject/luafilesystem/archive/v1_7_0_2.zip
- http://foicica.com/hg/gtdialog/archive/db67f8a489e8.zip
- http://invisible-mirror.net/archives/cdk/cdk-5.0-20150928.tgz
- https://foicica.com/hg/bombay/archive/tip.zip>bombay.zip
- http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.20.tar.gz"
-checksum="33d5c2eb36c93f48af2fa9fe6ae81ab2592b95af2fbe4f45144cd5b6d8a34ce0
- 40ee221a1c7bd827c1d7fd0fa7f41a6fddaa9bdf15667fc492b1d91a77681d57
- 0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac
- 48d66576051b6c78388faad09b70493093264588fcd0f258ddaab1cdd4a15ffe
- 9174ab891f1d392b42a587b2afb737f9565ecf8a860d2e5a2bff457858f58478
- 508be36c3ae0de3cb3eeddad9b8cb5cf31ff4c6e8aa9acf72c4e72d94e8e1ee3
- 028da75d5f777a1c4184f88e34918bd273bd83bbe3c959bc11710c4f0ea2e448
- 4d1ae87bc817302127b250a5968bd05d3f1599e2b9eb785f339698db0bb9294f
- 6c0d87c94ab9915e76ecd313baec08dedf3bd56de83743d9aa923a081935d2f5"
-skip_extraction="
- 99fa62b828ee.zip
- lua-5.3.5.tar.gz
- lpeg-1.0.2.tar.gz
- v1_7_0_2.zip
- db67f8a489e8.zip
- cdk-5.0-20150928.tgz
- bombay.zip
- libtermkey-0.20.tar.gz
-"
+distfiles="https://github.com/orbitalquark/textadept/archive/textadept_${version}.tar.gz
+	https://www.scintilla.org/scintilla445.tgz
+	https://github.com/orbitalquark/scinterm/archive/6a774158d8a3c7bc7ea120bc01cdb016fa351a7e.zip
+	https://github.com/orbitalquark/scintillua/archive/scintillua_4.4.5-1.zip
+	https://www.lua.org/ftp/lua-5.3.5.tar.gz
+	http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz
+	https://github.com/keplerproject/luafilesystem/archive/v1_8_0.zip
+	https://github.com/orbitalquark/gtdialog/archive/64587546482a1a6324706d75c80b77d2f87118a4.zip
+	https://invisible-mirror.net/archives/cdk/cdk-5.0-20200923.tgz
+	http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.20.tar.gz"
+checksum="3d2ef169465dea85ce0b8bfa19522e96006abcb043cb58c3c3b74ca1bec9b9df
+	4f2168684b4024dc3d6b267320fff1d729192047f4a1b79aeab9ec7c64f733ec
+	3b1e3cc3ee48bb158d1d666c7c86d190905c973918914876f6ab3712f8dd7d20
+	045c1a9c7e8eb356a7cc54ddd49e994fd2d33469ab34385f2f794fd1e2babc24
+	0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac
+	48d66576051b6c78388faad09b70493093264588fcd0f258ddaab1cdd4a15ffe
+	e3a6beca7a8a90522eed31db6ccdc5ed65a433826500c6862784e27671b9e18a
+	093a4f973196083610c4489e8d9272f340ebf82d48c064f1542c6464eda2af82
+	007f5de880cb2eebd8556df7e4cd8673d5e64c9970147eee6923a814c29faaed
+	6c0d87c94ab9915e76ecd313baec08dedf3bd56de83743d9aa923a081935d2f5"
+
+skip_extraction="scintilla445.tgz
+	6a774158d8a3c7bc7ea120bc01cdb016fa351a7e.zip
+	scintillua_4.4.5-1.zip
+	lua-5.3.5.tar.gz
+	lpeg-1.0.2.tar.gz
+	v1_8_0.zip
+	64587546482a1a6324706d75c80b77d2f87118a4.zip
+	cdk-5.0-20200923.tgz
+	libtermkey-0.20.tar.gz"
```

Comment:
Would it be possible to create variables for these versions, so you only have to fix them in a single place?

Also, `skip_extraction` here should be indented with spaces, not tabs.